includes/controllers/pages/class-submit-listing.php

```diff
@@ -32,7 +32,13 @@ public function get_title() {
         return __( 'Add Listing', 'business-directory-plugin' );
     }
 
-    public function enqueue_resources() {
+    /**
+     * Load custom resources used for this view only.
+     * This is called in the parent class `enqueue_resources` function.
+     *
+     * @since x.x
+     */
+    public function enqueue_custom_resources() {
```

Comment:
This function name was changed. Since it's a public function, we can't just remove it unless we deprecate the previous function.